Jenkinsfile

```diff
@@ -2,15 +2,19 @@ wrapperConfigurations = [
   Debug: 'dbg',
   Release: ''
 ]
-configuration = 'Release'
+configuration = 'Debug'
 
 xbuildCmd = '/usr/local/bin/xbuild'
-def nuget = '/usr/local/bin/nuget'
+nugetCmd = '/usr/local/bin/nuget'
+def windowsNugetCmd = 'C:\\ProgramData\\chocolatey\\bin\\NuGet.exe'
```

Comment:
Yes - it requires a chocolatey install of nuget on our CI slaves. I'm not sure if we have documentation about CI setup anywhere. Ping @fealebenpae.